5.5/run-mysqld-slave.sh

```diff
@@ -6,6 +6,7 @@ source ${HOME}/common.sh
 set -eu
 
 export MYSQL_RUNNING_AS_SLAVE=1
+export MYSQL_MASTER_ADDR=${MYSQL_MASTER_ADDR:-mysql-master}
```

Comment:
@bparees resolving the DNS in namespace is magic to me... I remember Clayton was explain this to us once but I can't remember the technical details. From what I remember is that it is safe to assume to DNS lookup for `mysql-master` will give you the IP of the service running in the current namespace.

If you want to be more precise, you can also include the namespace name in the DNS query (mysql-master.NAMESPACE) but I don't think we have NAMESPACE variable available in the container.